knowledge-base/anchore/sbom-action/action-security.yml

```diff
@@ -0,0 +1,10 @@
+name: "Anchore SBOM Action"
```

Comment:
Resolves #194

Reply:
@h0x0er can you please add link to reason (where it is mentioned that this needs artifacts access), like you did for `static-web-apps-deploy`? You can add it as a comment for `actions-reason`.

Reply:
@varunsh-coder thanks for pointing it. In the upcoming PRs I will make sure to add `reference links`.
For this one below are the links:
*   https://github.com/anchore/sbom-action/blob/3626d7d7b13e87ee6c6f9ded3940dea05a3967bc/src/github/SyftGithubAction.ts#L262
 *  https://github.com/anchore/sbom-action/blob/3626d7d7b13e87ee6c6f9ded3940dea05a3967bc/src/github/SyftGithubAction.ts#L459
 * https://github.com/anchore/sbom-action/blob/3626d7d7b13e87ee6c6f9ded3940dea05a3967bc/src/github/SyftGithubAction.ts#L465

Reply:
@h0x0er upload release asset requires `contents: write` permission
https://github.com/anchore/sbom-action/blob/3626d7d7b13e87ee6c6f9ded3940dea05a3967bc/src/github/SyftGithubAction.ts#L465